test/mocha/01-api.js

```diff
@@ -0,0 +1,270 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {agent} = require('bedrock-https-agent');
+const fs = require('fs');
+const {httpClient} = require('@digitalbazaar/http-client');
+const path = require('path');
+
+const {readFile} = fs.promises;
+
+describe('httpClientHandler', () => {
+  const BASE_URL = `https://localhost:18443`;
+  it('configured route should work properly', async () => {
```

Comment:
The way these tests should be read is: "it _____". This would read as "it configured route should work properly" which could be improved. Can this be changed to better match the convention and be a bit more clear on what is being tested? My suggestion would be:

```suggestion
  it('should get success response from configured route', async () => {
```

Reply:
Now that I've read a bit further, I think the context for these tests should change (as suggested above) and this should change to:

```suggestion
  it('should return 200 status code', async () => {
```

Other tests should be updated accordingly.